qiling/core.py

```diff
@@ -133,9 +133,11 @@ def __init__(
         if self.log_dir != "" and type(self.log_dir) == str:
             _logger = ql_setup_logging_env(self)    
             self.log_file_fd = _logger
-        else:
-            self.log_dir = None     
-        
+        elif self.console == True:
+            _logger = ql_setup_logging_stream(self)
+            self.log_file_fd = _logger
+            # self.log_dir = None
```

Comment:
this need to be  out.

Reply:
you mean the commented part or whole section ?

Reply:
yes comment one. unless there is a specific reason. if year please write it down